docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -86,6 +86,22 @@ The following example shows how to create a scalar Pandas UDF that computes the
 </div>
 </div>
 
+### Scalar Iterator
+
+Scalar iterator (`SCALAR_ITER`) Pandas UDF is the same as scalar Pandas UDF above except that the
+underlying Python function takes an iterator of batches as input instead of a single batch and 
+it yields output batches instead of returning a single output batch.
+It is useful when the UDF execution requires initializing some states, e.g., loading an machine
+learning model file to apply inference to every input batch.
+
```

Comment:
Add a note that the UDF can also return an iterator (instead of write a function with yield), and the input iterator is allowed to be prefetched.

Reply:
Updated.